modules/ROOT/pages/cors-policy.adoc

```diff
@@ -60,25 +77,47 @@ The public resource configuration looks like this:
 </http:listener-interceptors>
 ----
 
-If you cannot implement CORS for your API, another possible solution to an unreachable API is to disable the same-origin restrictions in your browser.
+== Disabling Same-Origin Restrictions in Your Browser
+
+If you cannot implement CORS for your API, you can disable the same-origin restrictions in your browser to call an inaccessible API. Each browser handles these restrictions in a unique way. You can research the tips, tricks, and plugins for your browser from the internet.
+
+Before you modify your browser settings, make sure you understand and assess its potential security implications. You must use these browser settings only for testing in your own web pages. Otherwise, the browser can become vulnerable to malicious scripts and other potential threats.
+
+=== Disabling Same-Origin Restrictions on Google Chrome for Mac OS X
+
+To disable same-origin restrictions on Google Chrome for Mac OS X:
+
+* Open a new Terminal window and paste the following line: `open -a Google\ Chrome --args --disable-web-security`.
```

Comment:
Paste from where? And don't they need to copy first?
Steps:
```suggestion
. Open a new Terminal window and paste within it the following line: 
`open -a Google\ Chrome --args --disable-web-security`
```

Reply:
Done! Thanks!